frontend/src/components/maps/leaflet/Map.scss

```diff
@@ -1,8 +1,8 @@
 .leaflet-container {
   /* Subtract the header, nav-bar and footer sizes */
-  min-height: calc(100vh - 70px - 45px - 42px);
-  margin-top: 10px;
-  padding-top: 20px;
+  height: calc(100vh - 70px - 45px - 42px - 76px);
```

Comment:
The map is now under the filter search bar so we need to adjust the height accordingly. Would love to find a way to improve this calculation here. Any ideas?